contracts/wrapper/NameWrapper.sol

```diff
@@ -1005,22 +1026,30 @@ contract NameWrapper is
     }
 
     function _canFusesBeBurned(bytes32 node, uint32 fuses) internal pure {
+        // If a non-parent controlled fuses is being burned, check PCC and CU are burnt
         if (
-            fuses & ~PARENT_CANNOT_CONTROL != 0 &&
+            fuses & ~PARENT_CONTROLLED_FUSES != 0 &&
             fuses & (PARENT_CANNOT_CONTROL | CANNOT_UNWRAP) !=
             (PARENT_CANNOT_CONTROL | CANNOT_UNWRAP)
         ) {
             revert OperationProhibited(node);
         }
     }
 
-    function _checkForParentCannotControl(bytes32 node, uint32 fuses)
+    function _checkForParentControlledFuses(bytes32 node, uint32 fuses)
         internal
         view
     {
-        if (fuses & PARENT_CANNOT_CONTROL != 0) {
-            // Only the parent can burn the PARENT_CANNOT_CONTROL fuse.
+        // Only the parent can burn the parent controlled fuses.
+        if (fuses & PARENT_CONTROLLED_FUSES != 0) {
             revert Unauthorised(node, msg.sender);
         }
     }
+
+    function _checkDotEthFuse(bytes32 node, uint32 fuses) internal pure {
```

Comment:
I'm not sure that would work in this situation. `IS_DOT_ETH` should not be settable at all by anyone otherwise someone could spoof a domain being .eth and access logic that is only meant for .eth names